### DISCUSSION TITLE
Hi guys, Can anyone tell me how to generate URI for V2 for Pairing.

### DISCUSSION THREAD

**Discussion by muzammalabdulghafoor000** [WalletConnect/walletconnect-monorepo#1123] [Developer Support]
I want to make a button in android app and when I click on it, then wallet connect UI should be open where I can connect my app to wallet. How I can achieve this? Any help or code snipped would be appreciated. Thanks

**Comment by chadyj** [comment]
Did you take a look at the kotlin v2 dapp sample app?
https://github.com/WalletConnect/WalletConnectKotlinV2#sample-apps
